components/tidb_query_datatype/src/codec/mysql/enums.rs

```diff
@@ -186,14 +190,14 @@ impl<'a> ToString for EnumRef<'a> {
 pub trait EnumEncoder: NumberEncoder {
     #[inline]
     fn write_enum(&mut self, data: EnumRef) -> Result<()> {
-        self.write_u64_le(data.value as u64)?;
+        self.write_u64_le(*data.value as u64)?;
```

Comment:
I forgot why I wrote `as u64` here😂. I think we can remove them.

Reply:
I've removed that :)